packages/ember-cli-fastboot/addon/services/fastboot.js

```diff
@@ -88,6 +89,11 @@ const FastBootService = Service.extend({
     return RequestObject.create({ request: get(this, '_fastbootInfo.request') });
   }),
 
+  _fastbootInfo: computed(function() {
```

Comment:
Do we need the test changes in https://github.com/ember-fastboot/ember-cli-fastboot/pull/823/files? Specifically registering `info:-fastboot` in the container for later lookup.

Reply:
It doesn't look like it based on the tests!  Do you see other value here?